tasks/Task.cpp

```diff
@@ -46,6 +47,10 @@ bool Task::configureHook()
     MagneticCalibration calibration = _magnetic_calibration.get();
     driver->writeMagneticCalibration(calibration);
 
+    constexpr auto rtk_parameter = findParameter("rtk-heading-to-mag-heading");
+    driver->writeConfiguration(rtk_parameter->index,
+        conf.rtk_heading2mag_heading.getDeg(), true);
```

Comment:
If you *really* want to avoid having a specific write, make the index a compile-time constant (and use the list of compile-time constants in `Parameters.?pp`) `findParameter` is essentially a compile-time constant that you search using a string in a list ...

Reply:
I don't see the gain since the result of findParamter is  already computed at compile-time. What can't I see?

Reply:
Me and @doudou have talked in peson and the argued that computing even in compile-time that we declared ourselves makes no sense to him, so I complied